src/components/QuickOpenModal.js

```diff
@@ -286,6 +292,51 @@ export class QuickOpenModal extends Component<Props, State> {
     return results && results.length ? results.length : 0;
   };
 
+  fuzzyhighlight = (query: any, results: ?Array<QuickOpenResult>) => {
+    const fullRes = [];
+
+    if (results) {
+      query = query.toLowerCase();
+      const queryUpper = query.toUpperCase();
+
+      let resultElem = 0;
+
+      for (resultElem; resultElem < results.length; resultElem++) {
+        let queryElem = 0;
+        if (query[0] === "@" || query[0] === "#" || query[0] === ":") {
+          queryElem++;
+        }
+
+        let titleElem = 0;
+        const title = results[resultElem].title;
+        const styledResults: Array<any> = [];
+
+        results[resultElem].title = "";
+
+        for (queryElem; queryElem < query.length; queryElem++) {
+          for (titleElem; titleElem < title.length; titleElem++) {
+            const q = query[queryElem];
+            const t = title[titleElem];
+            if (q === t || queryUpper[queryElem] === t) {
+              styledResults.push(
+                <span key={q + titleElem} style={styles}>
```

Comment:
Maybe something like `className="match"` instead of `style={styles}` so css can take care of it. If `.match` is not specific enough to trigger the css we can always amp up the specificity in the css file until it sticks, like ".result-list .result-item .title .match".

Reply:
This was very helpful, thanks. I got it working with your advice.